.travis.yml

```diff
@@ -12,3 +12,4 @@ script:
   - flake8 readit setup.py
   - python setup.py install
```

Comment:
@shitalmule04  
Please remove these lines because now you have done this test.
```
python setup.py install
readit --version
```

Reply:
@ganeshhubale, done.